crates/wasi-nn/examples/classification-component-onnx/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "classification-component-onnx"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
+
+[package.metadata.component]
+package = "component:classification-component-onnx"
+adapter = "../../../../target/wasm32-unknown-unknown/release/wasi_snapshot_preview1.wasm"
+
+[package.metadata.component.target.dependencies]
+"wasi:cli" = { path = "../../../wasi/wit/deps/cli" }
+"wasi:io" = { path = "../../../wasi/wit/deps/io"}
+"wasi:clocks" = { path = "../../../wasi/wit/deps/clocks"}
+"wasi:filesystem" = { path = "../../../wasi/wit/deps/filesystem"}
+"wasi:sockets" = { path = "../../../wasi/wit/deps/sockets"}
+"wasi:random" = { path = "../../../wasi/wit/deps/random"}
+"wasi:nn" = { path = "../../spec/wit/wasi-nn.wit" }
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+cargo-component-bindings = { git="https://github.com/bytecodealliance/cargo-component" }
+image = { version = "0.24.6", default-features = false, features = ["gif", "jpeg", "ico", "png", "pnm", "tga", "tiff", "webp", "bmp", "hdr", "dxt", "dds", "farbfeld"]  }
+ndarray = "0.15.3"
```

Comment:
I don't know how easy it will be to add these dependencies to `test-programs`. If it is difficult for some reason, you could always dump the decoded bytes to a file like I did with the OpenVINO test fixtures. If you do keep this, though, we probably only need the `jpeg` feature.

Reply:
added to test-programs and reduced to jpeg.

Reply:
In case some Git mischief happened here: `image` looks to be pulling in all the features again, not just `jpeg`.